[BUGFIX] Envoyer l'assessmentId lorsqu'on modifie une certification (PIX-1421)

Anne-Gaelle-S opened the pull request:
## :unicorn: Problème
Sur Pix Admin lorsqu'on modifie une certification au status started et qu'on enregistre cela n'enregistre pas les changements.

En effet, lorsqu'on modifie une certification un nouveau assessmentResult est créé.
Ici on ne passait pas à la requête l'assessmentId, ce qui avait pour conséquence de créer un assessmentResult sans assessmentId et on ne pouvait pas observer les changements sur PixAdmin.
Cela était du au fait que lorsqu'on récupérait les résultats d'une certification on n'obtenait pas l'assessmentId associé (car la certif étant restée au status "started").

## :robot: Solution
Aller chercher l'assessmentId à partir du certifCourseId lorsque la certif est restée en "started" (cad quand la certification n'a pas d'assessemnt-result encore).

Le premier commit permet au front de récupérer une erreur plus parlante dans le front en cas de soucis : 
<img width="1395" alt="image" src="https://user-images.githubusercontent.com/38167520/96134201-7f48f500-0efb-11eb-9fd3-7c459e413e8a.png">

Le deuxième commit répare le bug.

## :rainbow: Remarques
> _Des infos supplémentaires, trucs et astuces ?_

## :100: Pour tester
Sur mon-pix : 
- lancer un test de certif avec n'importe qui et ne répondre à aucune question (2, anne, success, 01/01/2000, ANNE02)
- noter son numéro de certif
Sur admin : 
- aller dans l'onglet certification et mettre l'id de certif noté précédemment
- constater que l'état de la certif est en "started" dans les infos
- aller dans l'onglet "Détail"
- scroller en bas + cliquer sur "enregistrer"
- scroller en haut, modifier le status de "started" à "validated"
- scroller en bas + cliquer sur "enregistre"
- rafraîchir la page et constater que le status de la certif est bien changé (validated)

Anne-Gaelle-S committed:
[API] Throw in repo to get error

pix-service commented:
I'm deploying this PR to these urls:

- App: https://app-pr2013.review.pix.fr
- Orga: https://orga-pr2013.review.pix.fr
- Certif: https://certif-pr2013.review.pix.fr
- Admin: https://admin-pr2013.review.pix.fr
- API: https://api-pr2013.review.pix.fr/api/

 Please check it out!

Anne-Gaelle-S committed:
[API] BUGFIX return assessmentId

When the certif is started lastAssessmentResult is null
So we must return the correct assessmentId in this case
We can get if from certificationCourseId
